components/docs-chef-io/content/automate/major_upgrade_4.x.md

```diff
@@ -0,0 +1,315 @@
++++
+title = "Upgrade to Chef Automate 4.x"
+date = 2022-03-03T12:02:46-08:00
+draft = false
+gh_repo = "automate"
+
+[menu]
+  [menu.automate]
+    title = "Upgrade to 4.x"
+    identifier = "automate/upgrade/major_upgrade.md Major Upgrade 4.x"
+    parent = "automate/upgrade"
+    weight = 20
++++
+
+Chef Automate provides an entire suite of enterprise capabilities for node visibility and compliance. Chef Automate upgrades from one minor version to another automatically. However, Chef Automatate will not automatically upgrade to a major version. See the instructions below for manually upgrading Chef Automate from date-based versions to Chef Automate 4.x.
+
+## Upgrade Journey
+
+Please choose following upgrade journey based on your current version of Chef Automate. All of these upgrades are manual upgrades.
+
+| Your Current Version | Upgrade To |
+| -------------------- | ---------- |
+| Any version before 20220329091442| 20220329091442|
+| 20220329091442| 3.0.x|
```

Comment:
I believe it's not 3.0.x, it should be a milestone version of 3.x.x

Reply:
done